sdk/core/azure-core/azure/core/_pipeline_client.py

```diff
@@ -204,9 +193,9 @@ def send_request(self, request, **kwargs):
         :return: The response of your network call. Does not do error handling on your response.
         :rtype: ~azure.core.rest.HttpResponse
         # """
-        rest_request, request_to_run = _prepare_request(request)
+        rest_request = hasattr(request, "content")
```

Comment:
when the requests had a different surface area than the old requests, we had to convert it first to pass it through the pipelines. Now, we can directly pass through

Reply:
Do we still need it?